hw/top_earlgrey/ip/sensor_ctrl/data/sensor_ctrl.hjson

```diff
@@ -0,0 +1,166 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+// TODO: This module is only a draft implementation that covers most of the ast_wrapper
+// functionality but is incomplete
+
+
+# SENSOR_CTRL register template
+#
```

Comment:
Didn't know that we can use `#` for comment.

Reply:
@tjaychen 
ast_wrapper looks good.
please explain the temp change in aon_osc freq.